ethy-gadget/src/keystore.rs

```diff
@@ -85,6 +85,7 @@ impl EthyKeystore {
 	/// Use the `public` key to verify that `sig` is a valid signature for `message`.
 	///
 	/// Return `true` if the signature is authentic, `false` otherwise.
+	#[allow(dead_code)]
```

Comment:
Why is this necessary?

Reply:
suppresses the compiler warning about the function being unused (more OCD than anything important)